inference-engine/src/gna_plugin/frontend/layer_quantizer.hpp

```diff
@@ -91,6 +91,11 @@ struct QuantI8_I8 : public QuantDescTmpl<PRECISION_TYPE(I8, I32, I8, I8, MIXED)>
 };
 
 // for support proper trait instantiation for quantization function callback
+struct FakeQuant : public QuantDescTmpl<P_TYPE(I16), P_TYPE(I32), P_TYPE(MIXED), P_TYPE(MIXED), P_TYPE(MIXED)> {
```

Comment:
BTW, I don't see where network precision is used (`getNetPrecision` is never called). Can we remove `_Np` at all? There is no sense in it. Network precision is always MIXED.

Reply:
removed